src/modules/git_commit.rs

```diff
@@ -40,6 +40,39 @@ pub fn module<'a>(context: &'a Context) -> Option<Module<'a>> {
         )),
     );
 
+    if !config.tag_disabled {
+        // Let's get repo tags names
+        let tag_names = git_repo.tag_names(None).unwrap();
```

Comment:
I'm not sure under what condition the `unwrap()` would panic, but it may be best if we handled it as though no tag names are present. I would rather we show less data than break the prompt.

Reply:
Thank you so much for your time reviewing this PR, @matchai. And sorry for request directly your review. I noticed I should'n later, reading the whole [CONTRIBUTING](https://github.com/starship/starship/blob/master/CONTRIBUTING.md#gitgithub-workflow) document... :man_facepalming: